src/__tests__/useAsyncMemoSuspense.test.tsx

```
@@ -0,0 +1,370 @@
+    </div>
+  );
+}
+
+// Error boundary for testing error scenarios
+interface ErrorBoundaryState {
+  hasError: boolean;
+  error: Error | null;
+}
+
+class ErrorBoundary extends React.Component<
+  React.PropsWithChildren<{}>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)